cmd/apps/openfaas_app.go

```diff
@@ -229,10 +229,18 @@ func getValuesSuffix(arch string) string {
 
 func mergeFlags(existingMap map[string]string, setOverrides []string) error {
 	for _, setOverride := range setOverrides {
-		flag := strings.Split(setOverride, "=")
+		flag := strings.SplitN(setOverride, "=", 2)
 		if len(flag) != 2 {
 			return fmt.Errorf("incorrect format for custom flag `%s`", setOverride)
 		}
+		isStartFlagsParsed := false
```

Comment:
What does `isStartFlagsParsed` mean? I think there may be a better variable name that explains what this is doing.

Reply:
Yeah at that point of time, a thought this is a-okay, now reading only this 3 rows I honestly have hard time to figure it out what this does. We've all been there :D I'll change this ASAP.